voting/set/right/vote_count.rb

```diff
@@ -181,24 +274,4 @@ def down_details
       }
     end
   end
-
-  view :details do |args |
-    wrap args.merge(:slot_class=>'nodblclick') do
-      [
-        wrap_with_class('vote-up') do
-          [
-            vote_up_link(:details),
-            up_details
-          ]
-        end,
-        _render_core( args ),
-        wrap_with_class('vote-down') do
-          [
-            vote_down_link(:details),
-            down_details
-          ]
-        end
-      ]
-    end
-  end
 end
```

Comment:
no change in HtmlFormat, only reorganized. I moved all views to the top and helper methods to the bottom, extracted the method vote_path and replaced `wrap_with_class` with `wrap_with`